app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,9 @@
+class WelcomeController < ApplicationController
+  def show
+    if !current_user || params[:id] == 'landing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.